client/src/components/sources/sources.js

```diff
@@ -280,7 +311,14 @@ class Sources extends React.Component {
         />,
         <Grid fluid key={2}>
           {this.renderList(filteredItems)}
-        </Grid>
+        </Grid>,
+        <CreateScanDialog
+          key="createScanDialog"
+          show={scanDialogShown}
+          sources={multiSourceScan ? selectedItems : [currentScanSource]}
+          onCancel={this.hideScanDialog}
+          onScan={this.createScan}
+        />
```

Comment:
Per above, the support component is missing... Commenting the reference for the component and removing this block makes the app run again...